packages/payment-integration-api/src/payment/payment.ts

```diff
@@ -3,14 +3,14 @@ import { Omit } from '../util-types';
 
 import PaymentAdditionalAction from './payment-additional-action';
 
-export default interface Payment {
+export default interface Payment<FP = unknown> {
```

Comment:
Why FP?

Reply:
FP - is like formatted payload type

Reply:
Seems fine to name it `T` even since it's a generic. But this line confuses me then https://github.com/bigcommerce/checkout-sdk-js/pull/2366/files#diff-5c53fae9c62715282b70567f8e100092944dc0f3335e998b9b38fd7dacf0f77eR44. We passing a generic, but it can be it's own type as well.